src/components/admin/AdminGlobalMute.vue

```diff
@@ -158,6 +158,7 @@ export default defineComponent({
 
     const globallyMutedPlayers = computed<GloballyMutedPlayer[]>(() => adminStore.globallyMutedPlayers);
     const banDateSet = computed<boolean>(() => banExpiry.value != "");
+    const author = computed<string>(() => oauthStore.blizzardVerifiedBtag);
```

Comment:
What is this here for?

Lets make sure we retrieve and set the author on the website-backend; rather than trusting the input - just to avoid impersonation.

Reply:
Permission checking is done in the backend already. We trust one mod to not impersonate another.

Reply:
But why even bother introducing this as a property in the frontend if we can just retrieve it in the backend?

If we have the ability to retrieve it from the auth credentials, I'd definitely prefer that. Verify > trust.